src/main/java/edu/ucsb/ucsbcslas/controllers/ActiveQuarterController.java

```diff
@@ -52,11 +53,30 @@
     if (!authControllerAdvice.getIsAdmin(authorization))
       return getUnauthorizedResponse("admin");
     activeQuarterRepo.deleteAll();
-    
+
+    String quarterId = "";
+    String quarterYear = "20" + String.valueOf(activeValue.substring(1));
+
+    if(activeValue.charAt(0) == 'W'){
+      quarterId = "1";
+    }
+    else if(activeValue.charAt(0) == 'S'){
+      quarterId = "2";
+    }
+    else if(activeValue.charAt(0) == 'M'){
+      quarterId = "3";
+    }
+    else if(activeValue.charAt(0) == 'F'){
+      quarterId = "4";
+    }else{
+      throw new IllegalArgumentException("Invalid argument");
+    }
```

Comment:
Consider using [indexOf](https://www.w3schools.com/java/ref_string_indexof.asp)

```
quarterId = "_WSMF".indexOf(activeValue.charAt(0));
if (quarterId < 1)
   throw new IllegalArgumentException("Invalid argument");
```

14 lines of code becomes 3.